CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -246,4 +246,43 @@ public static function getFinancialAccountForFinancialTypeByRelationship($financ
     return Civi::$statics[__CLASS__]['entity_financial_account'][$financialTypeID][$relationTypeId];
   }
 
+  /**
+   * Get Financial Account type relations
+   *
+   * @param $flip bool
```

Comment:
@pradpnayak it appears to me that $flip is functionally equivalent to calling function calling array file and your explanation about key and value of $financialAccountLinks illustrates that - so I agree with @eileenmcnaughton . Please update to take account of her suggestion.